example/app/src/main/java/co/moonmonkeylabs/realmrecyclerview/example/MainActivity.java

```diff
@@ -359,15 +357,12 @@ protected Void doInBackground(Void... params) {
                 // Add some delay to the refresh/remove action.
                 try {
                     Thread.sleep(1000);
-                } catch (InterruptedException e) {
+                } catch (InterruptedException ignored) {
                 }
                 Realm instance = Realm.getInstance(getRealmConfig());
-                final RealmResults<QuoteModel> all = instance.where(QuoteModel.class).findAll();
-                if (all != null) {
-                    instance.beginTransaction();
-                    all.clear();
-                    instance.commitTransaction();
-                }
+                instance.beginTransaction();
+                instance.delete(QuoteModel.class);
```

Comment:
Good change. Yea, I think this should be fine / I think I read that the functionality of clear on a realmResult list changed. 

👍